src/containers/profile/profileTab/styles.module.scss

```diff
@@ -0,0 +1,144 @@
+@import '../../../scss/shared/breakpoints.scss';
```

Comment:
Duplicated code, try to see how to use @media query from another repo for instance